faiss_vector_posting.go

```diff
@@ -310,7 +310,7 @@ func (sb *SegmentBase) InterpretVectorIndex(field string, requiresFiltering bool
 	// Params needed for the closures
 	var vecIndex *faiss.IndexImpl
 	var vecDocIDMap map[int64]uint32
-	var docVecIDMap map[uint32][]int64
+	var docVecIDMap map[int64][]uint32
```

Comment:
Why the change in type for the key in this map?

Reply:
good point, it can remain uint32.